manifests/profile/server.pp

```diff
@@ -390,9 +395,9 @@
     }
 
     St2::Package::Install<| tag == 'st2::profile::server' |>
-    -> Ini_setting<| tag == 'st2::profile::server' |>
+    -> Ini_setting<| tag == 'st2::config' |>
     -> Exec['start st2']
```

Comment:
This part is actually defunct. We use the `ng_init` pattern, so it will do an actual service restart.